platforms/database.go

```diff
@@ -0,0 +1,93 @@
+package platforms
+
+import (
+	"runtime"
+	"strings"
+)
+
+// validComboString defines valid GOOS/GOARCH combinations.
+// Should be updated from https://golang.org/doc/install/source#environment.
+// This is why we leave it as a string.
+const (
+	validComboString = `
```

Comment:
@AkihiroSuda We are going to support the full Go space for platform declaration. This does not imply actual support. This is just identification.